src/dotnet-install.ps1

```
@@ -711,7 +711,7 @@ $installDrive = $((Get-Item $InstallRoot).PSDrive.Name);
 $diskInfo = Get-PSDrive -Name $installDrive
 if ($diskInfo.Free / 1MB -le 100) {
     Say "There is not enough disk space on drive ${installDrive}:"
-    exit 0
+    return
```

It seems that there should be "throw" instead of the "return". 
And here we also need to do "Say-Error" instead of "Say"

---

Maybe just throw that message?

---

Let's do all error-related changes in other PR as agreed on stand up